firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandler.cs

```diff
@@ -60,6 +60,34 @@ public class UIHandler : MonoBehaviour {
     // Previously completed task.
     protected Task previousTask;
 
+    private enum Backend {
+      Production = 0,
+      Emulator = 1
+      // Nightly and stage can be added later.
+    }
+
+    private Backend TargetBackend = GetTargetBackend();
+    // For local testing purpose, manually modify the port number if Firestore
+    // emulator is running at a different port.
+    private string EmulatorPort = "8080";
+
+    private static Backend GetTargetBackend() {
+      // Set Firestore emulator as default backend.
+      Backend targetBackend = Backend.Emulator;
```

Comment:
I think it would make more sense to use Production as the default, since additional steps are needed to set up the Emulator

Reply:
I am a little confused by changes. It appears you are starting emulator as part of CI now. Does that mean we are NOT testing against prod anymore?

Reply:
Discussed offline. CI tests on UNITY_EDITOR (playmode) or UNITY_STANDALONE(macOS, Linux, Windows) are default to run against Emulator. And other platforms(iOS, tvOS, Android) will run against prod.

Reply:
My major concern is that someone will try to run the Firestore testapp locally in editor without knowing that they have to also spin up an Emulator, and then get a bunch of errors.  But if you all are fine with that though, that's fine.

Reply:
While running Firestore tests against the emulator, it will print a line of "Running tests against Firestore Emulator.". Hopefully, this would give the users some hint.